spec/03-plugins/09-key-auth/02-access_spec.lua

```diff
@@ -259,7 +282,7 @@ for _, strategy in helpers.each_strategy() do
           }
         })
         res:read_body()
-        assert.equal('Key realm="' .. meta._NAME .. '"', res.headers["WWW-Authenticate"])
+        assert.equal('Key', res.headers["WWW-Authenticate"])
```

Comment:
That's correct. The plugin instance resolved on `key-auth1.com` doesn't have `realm` set and since it's optional for everything except `basic-auth` then we don't want to add it. This diff shows that we don't include `realm="something"` unless it's been explicitly set